ui/package.json

```
@@ -59,6 +59,7 @@
     "@babel/plugin-proposal-decorators": "^7.21.0",
     "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
     "@babel/plugin-transform-block-scoping": "^7.12.1",
+    "@ember-data/legacy-compat": "^5.2.0",
```

Found in embroider test branch -- this version should match our ember-data version 
```suggestion
    "@ember-data/legacy-compat": "~4.12.4",
```